recipe/meta.yaml

```diff
@@ -25,108 +25,43 @@ requirements:
   run:
     - aiohttp >=3.8.3,<4.0.0
     - async-timeout >=4.0.0,<5.0.0  # [py<311]
-    - dataclasses-json >=0.5.7,<0.7.0
-    - jsonpatch >=1.33,<2.0
-    - langchain-community >=0.0.32,<0.1
-    - langchain-core >=0.1.41,<0.2.0
-    - langchain-text-splitters >=0.0.1,<0.1
+    - langchain-core >=0.3.21,<0.4.0
+    - langchain-text-splitters >=0.3.0,<0.4.0
     - langsmith >=0.1.17,<0.2.0
-    - numpy >=1.0.0,<2.0.0
-    - pydantic >=1.0.0,<3.0.0
+    - numpy >=1.22.4,<2.0.0  # [py<312]
+    - numpy >=1.26.2,<3.0.0  # [py>=312]
+    - pydantic >=2.7.4,<3.0.0
     - python
     - pyyaml >=5.3
     - requests >=2.0.0,<3.0.0
     - sqlalchemy >=1.4,<3
-    - tenacity >=8.1.0,<9.0.0
+    - tenacity >=8.1.0,!=8.4.0,<9.0.0
   run_constrained:
```

Comment:
I would remove all the run_constrained section. 
- The current constraints are not matching upstream. 
- It is not clear if these are dependencies extending the funcitonality of langchain, or dependencies used at testing time only (in which case they should go in test/requires, if we'd enabled the tests.
- Constraints copied from, for example langchain-openai would be enforced by installing langchaoin-openai by a customer.

Reply:
Agreed. I've removed whole section.